starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +17,40 @@ var words = [
   'crackpot'
 ];
 
+
+function findLongestWord(arr){
+  return arr.reduce(function(acc,val){
```

Comment:
Good aproach, but there is a problem. This code doesn't work for an empty array (is one of the tests). You can start checking if the array is empty (and then returning undefined) or include an initial value of undefined:
```
function findLongestWord(arr){
  return arr.reduce(function(acc,val){
    if(val.length>acc.length){
      acc=val;
    }
    return acc;
  }, undefined)
}
```